src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -11,6 +11,33 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
+    public boolean equals(Rectangle rectangle) {
+        if (this == rectangle) {
+            return true;
+        }
+        if (rectangle == null) {
+            return false;
+        }
+        if (rectangle.getClass().equals(Rectangle.class)) {
+            return ((width == rectangle.width)
+                    || (width != null && width.equals(rectangle.width)))
```

Comment:
```suggestion
            return (width == rectangle.width
                    || width != null && width.equals(rectangle.width))
```
remove redundant braces;